app/src/main/res/layout-sw600dp/topic_practice_header_view.xml

```diff
@@ -9,7 +9,7 @@
       type="org.oppia.android.app.topic.practice.practiceitemviewmodel.TopicPracticeHeaderViewModel" />
   </data>
 
-  <FrameLayout
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
why are we using contraint layout instead of frame layout in this?

Reply:
We dont have frame layout in phone layout. I think the use of frame layout here was just to center the child constraint layout. The same can be done by using constriant layout inside constraint layout. This way we can eliminate the use of `android:layout_gravity` . Also say some comments by @rt4914 in previous PRs where he suggested to use constraint layout instead of frame layout